src/common.rs

```diff
@@ -1233,3 +1239,61 @@ mod tests {
         );
     }
 }
+
```

Comment:
nit: It's kind of weird to see these functions being defined after the test module. Usually everything should go before it just for consistency